ui/src/app/pages/workspace/research-purpose.tsx

```diff
@@ -51,8 +54,26 @@ const styles = reactStyles({
   sectionText: {
     fontSize: '14px', lineHeight: '24px', color: colors.primary, marginTop: '0.3rem'
   },
+  reviewPurposeReminder: {
+    marginTop: '0.3rem',
+    borderStyle: 'solid',
+    height: '2.5rem',
+    color: colors.primary,
+    alignItems: 'center',
+    justifyContent: 'center',
+    borderColor: colors.warning,
+    borderRadius: '0.4rem',
+    borderWidth: '0.1rem',
+    backgroundColor: colorWithWhiteness(colors.highlight, 0.7)
+  }
 });
 
+function updateWorkspace(workspace) {
+  workspace.researchPurpose.researchPurposeReviewed = true;
```

Comment:
Ideally, we would do this logic in the `WorkspaceService` with a dedicated endpoint just for updating whether the user has reviewed the research purpose.

Reply:
👍